### PR TITLE
Adding additional doc fragments to panos.py

### PR DESCRIPTION
**Pull request by shinmog**
##### SUMMARY

Adding additional documentation fragments for `panos` modules to use.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

panos.py

### CODE REVIEW

File: lib/ansible/plugins/doc_fragments/panos.py [hunk 1]
```diff
@@ -25,3 +25,221 @@ class ModuleDocFragment(object):
         type: str
         default: admin
 '''
+
+    PROVIDER = r'''
+options:
+    provider:
+        description:
+            - A dict object containing connection details.
+        version_added: 2.8
```
Comment (gundalow) [inline lib/ansible/plugins/doc_fragments/panos.py]:
```suggestion
        version_added: '2.8'
```

File: lib/ansible/plugins/doc_fragments/panos.py [hunk 2]
```diff
@@ -25,3 +25,221 @@ class ModuleDocFragment(object):
...
+                      this param is ignored.
+                type: str
+'''
+
+    TRANSITIONAL_PROVIDER = r'''
+options:
+    provider:
+        description:
+            - A dict object containing connection details.
+        version_added: 2.8
```
Comment (gundalow) [inline lib/ansible/plugins/doc_fragments/panos.py]:
```suggestion
        version_added: '2.8'
```

File: lib/ansible/plugins/doc_fragments/panos.py (hunk 1 above)
Comment (acozine) [reply lib/ansible/plugins/doc_fragments/panos.py]: Resolved in 0595acc4f78f55103e406b523b7fd67115105162

File: lib/ansible/plugins/doc_fragments/panos.py (hunk 2 above)
Comment (acozine) [reply lib/ansible/plugins/doc_fragments/panos.py]: Resolved in 0595acc4f78f55103e406b523b7fd67115105162

**Review by gundalow — APPROVED** [review]
Thanks!

**Pull request merged**